src/core/hiwire.c

```diff
@@ -388,17 +388,37 @@ EM_JS_REF(JsRef, hiwire_dir, (JsRef idobj), {
   return Module.hiwire.new_value(result);
 });
 
+static JsRef
+convert_va_args(va_list args)
+{
+  JsRef idargs = hiwire_array();
+  while (true) {
+    JsRef idarg = va_arg(args, JsRef);
+    if (idarg == NULL) {
+      break;
+    }
+    hiwire_push_array(idargs, idarg);
+  }
+  va_end(args);
+  return idargs;
+}
+
 EM_JS_REF(JsRef, hiwire_call, (JsRef idfunc, JsRef idargs), {
   let jsfunc = Module.hiwire.get_value(idfunc);
   let jsargs = Module.hiwire.get_value(idargs);
   return Module.hiwire.new_value(jsfunc(... jsargs));
 });
 
-EM_JS_REF(JsRef, hiwire_call_OneArg, (JsRef idfunc, JsRef idarg), {
-  let jsfunc = Module.hiwire.get_value(idfunc);
-  let jsarg = Module.hiwire.get_value(idarg);
-  return Module.hiwire.new_value(jsfunc(jsarg));
-});
+JsRef
+hiwire_call_va(JsRef idobj, ...)
```

Comment:
What does "va" stand for in the name? Could we use something more explicit?

OK I see http://www.cplusplus.com/reference/cstdarg/
Maybe then at least have the term "Variable arguments" appear somewhere in the docstring.

Reply:
> Could we use something more explicit?

Perfectly happy to if you have a suggestion. For what it's worth, the Python equivalents are `PyObject_CallFunctionObjArgs` and `PyObject_CallMethodObjArgs`. (I also think we would do well to move to mixed snake / camel case naming in hiwire like what Python does.)

Reply:
`hiwire_CallFunctionObjArgs` ?  I guess keeping the CPython C naming conventions would make sense.

But no strong feeling about it, we can also keep it as is and improve names in some other PR.